Add Empty Analysis Dir

roshankern opened the pull request:
This PR adds an empty analysis directory to Brieflow that others can use for their screens. We also improve documentation for how to use Brieflow. 

Addresses #22

roshankern commented:
@mat10d this is ready for review!

mat10d commented on README.md:
```diff
@@ -70,57 +79,82 @@ Use the following commands to set up the `brieflow_configuration` Conda environm
 
+### Example Analysis
 
-## Contribution Notes
```
The only thing I would add is some note about issues and that we're open to contributions at this early stage--and maybe some documentation that tells you how to do this?

roshankern replied:
Done in https://github.com/cheeseman-lab/brieflow/pull/28/commits/58f22adfa2b4afacbde5654716c84a01fcd2707a

mat10d commented on README.md:
````diff
@@ -34,21 +34,30 @@ workflow/
+git clone https://github.com/cheeseman-lab/brieflow.git
+```
+
+See the steps below to set up the workflow/configuration environments and run your own analysis with Brieflow.
````
I would add that we will soon release documentation on how to set up an analysis repo for working with brieflow!

roshankern replied:
Done in https://github.com/cheeseman-lab/brieflow/pull/28/commits/d2abbe5c2ac1a1c9fb5cadb8fefb50b59533d162

mat10d reviewed (commented):
Looks good to me. 

We may get a lot of suggestions from Bryce. It would be awesome if we can have him adding functions that would be useful for us. To this end--does this README, or does another README provide information on how

1. The directory is structured (with workflows, libs, scripts, snakemake rules, targets, etc?
2. How to submit issues
3. Where to add functionalities and how to submit PRs? -- to this end, providing some info on how to create a different branch would be good
4. Just a q--do we have some control on who can merge code? Because our soft launch this afternoon may open us up to other people fucking things up

mat10d commented:
Good to merge from my end @roshankern

roshankern commented:
> The directory is structured (with workflows, libs, scripts, snakemake rules, targets, etc?
Done in https://github.com/cheeseman-lab/brieflow/pull/28/commits/19111820341a8758e706f5325b9d36be24084ac8

roshankern commented:
> How to submit issues

Done in https://github.com/cheeseman-lab/brieflow/commit/40a9d0c162b975ba2f36576a848c5765c5a63d13

roshankern commented:
> Where to add functionalities and how to submit PRs? -- to this end, providing some info on how to create a different branch would be good

Added a PR template, included a comment to use this to add functionality to Brieflow, and added how to create fork, branch, pr.

roshankern commented:
> Just a q--do we have some control on who can merge code? Because our soft launch this afternoon may open us up to other people fucking things up

Collaborators (you and myself right now) can create branches on this repo. Contributors (anyone else) has to fork the repo, create a branch there, then add to this. Should help keep this repo clean and roles clear. I also added a rule that requires an approval by an admin before merging to the default branch (main).
